validation_server/validation_server.js

```diff
@@ -1,27 +1,33 @@
 const axios = require("axios");
-const { B3Propagator } = require("@opentelemetry/core");
-const { HttpTraceContext } = require("@opentelemetry/core");
-const { NodeTracerProvider } = require("@opentelemetry/node");
-const bodyParser = require("body-parser");
-const opentelemetry = require("@opentelemetry/api");
-
-// Get the args
-const myArgs = process.argv.slice(2);
+const {
+  HttpTraceContext,
+  getExtractedSpanContext,
 
-// Get the propagator type
-const propagatorTypeArg = myArgs[0];
-const myPropagator =
-  propagatorTypeArg === "b3" ? B3Propagator : HttpTraceContext;
+  setExtractedSpanContext,
+} = require("@opentelemetry/core");
```

Comment:
You require things from core, tracing, and context-base, but none of them are in your dependencies.